geoportailv3/static/js/catalog/layertree.html

```diff
@@ -1,5 +1,5 @@
-<ul id="catalog-{{::layertreeCtrl.uid}}" class="catalog">
-  <li ng-repeat="node in ::layertreeCtrl.tree.children"
```

Comment:
I think I understand why removing that one-time binding may be required, but could you please confirm that it doesn't work when using one-time binding?